main.go

```diff
@@ -35,6 +37,18 @@ func main() {
 			Value: "",
 			Usage: "specify the directory that cache data will be stored",
 		},
+		&cli.StringFlag{
+			Name:    "seed",
```

Comment:
@hsanjuan can you explain a bit more what you're hoping to do with the persistent identities here?

Examples:
- Debuggability/traceability
  - Note: if so you may also want the DHT client host to have a persistent key.... although practically I doubt that'd be very helpful.
- Ability for nodes to peer with rainbow so they can fetch blocks from it over Bitswap
   - This is currently disabled, is the plan to re-enable this (e.g. but limited only to a subset of nodes that we want to pay the egress for, such as within a cluster of rainbow nodes)
- Ability to symmetrically peer with nodes we want to fetch blocks from over Bitswap (e.g. so that nodes we're peered with know to like us/not prune our connections)

Reply:
It's explained above. Gateways should be peered among themselves and with others. This requires needing to track keys for each gateway, which are secrets and need special storage. Then need to track peer IDs for each gateway. Then need to track IPs and hostnames for each gateway. Then need to configure peering with all that information in a persistent file before the process starts.

That is all unecessary: a single secret can configure all nodes and they can derive peerIDs for other nodes on the fly. Since every node bootstraps, the dht can be used to find routing.

This allows deploying identical machines that peer between themselves without having to generate separate secrets and change the configuration of every other machine when that happens.